code/modules/shuttle/shuttle.dm

```diff
@@ -874,13 +874,16 @@
 	possible_destinations = null // Set at runtime
 
 /obj/machinery/computer/shuttle/white_ship/Initialize(mapload)
+	initialized = TRUE // DO NOT EVER EVER EVER DO THIS ANYWHERE ELSE I SWEAR TO GOD
```

Comment:
Please dont break my legs

Reply:
What's going on here?

Reply:
Can you add a reason to why it's here instead of just this warning?